linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/combined_client_graph.js

```diff
@@ -0,0 +1,52 @@
+var CombinedClientGraph = (function() {
+  function clientToMetric(client) {
+    return {name: client, color: ""}; //TODO: move to clientName only after v2 migration
+  }
+
+  return function(metricsCollector, routerName, $root, colors) {
+    var query = Query.clientQuery().withRouter(routerName).withMetric("requests").build();
+
+    var timeseriesParams = function(metricName) {
+      return {
+        strokeStyle: colors[colorIdx++ % colors.length],
+        lineWidth: 2
+      };
+    };
+
+    var chart = new UpdateableChart(
+      {
+        minValue: 0,
+        grid: {
+          strokeStyle: 'rgba(255,255,255,0.3)',
+          fillStyle: 'rgba(0,0,0,0)',
+          verticalSections: 1,
+          millisPerLine: 10000,
+          borderVisible: false
+        },
+        labels: {
+          fillStyle: '#878787',
```

Comment:
Totally minor, but why use rgba for the grid.fillStyle, and hex for the labels.fillStyle?